README.md

```diff
@@ -1,79 +1,21 @@
-# Base template for Gulp
+# Tree from object
 
-Template to start from before starting frontend projects.
+## Task
 
-## Install template
+Write a function `createTree` that creates a list of `ul/li` from an object.
```

Comment:
`createTree(element, data)` - to avoid the next line
`a nested list`

Reply:
changed